modules/gapi/src/executor/gstreamingexecutor.cpp

```diff
@@ -472,6 +499,7 @@ bool QueueReader::getResultsVector(std::vector<Q*>   &in_queues,
             // Collector thread never handles the inputs directly
             // (collector's input queues are always produced by
             // islands in the graph).
+            GAPI_LOG_INFO(nullptr, "try on theory...");
```

Comment:
Didn't get the log msg

Reply:
Comment on top
```
            // In theory, the CNST should never reach here.
            // Collector thread never handles the inputs directly
            // (collector's input queues are always produced by
            // islands in the graph).
```
I have suspicions about comments in code starting from "In theory..." therefor I suppose a weak in program logic in this particular place - so it should be wrapped in logger at least.

I agree that "try on theory..." is a bad message and must be rephrased - @dmatveev , @TolyaTalamanov  could you help me to produce a correct message here? because i do not understand "In theory..." comment

Reply:
> therefor I suppose a weak in program logic in this particular place - so it should be wrapped in logger at least.

Let me explain what it means. The code is written here in assumption that something never happens. This assumption is indeed true at the point when the code is written, but things change and years later someone can change something and break old contracts. We've had these examples already

Reply:
so the point is not to tell in comments anything for sure. This shouldn't be relevant to the logs you write, though.